COMMUNICATIONS.md

```diff
@@ -0,0 +1,95 @@
+# OpenSearch Project Communication 
+
+- [Overview](#overview)
+- [Slack](#slack)
+  - [Getting Started](#getting-started)
+  - [Workspace Channels](#workspace-channels)
+  - [Tips](#tips)
+
+## Overview
+
+The purpose of this document is to information regarding the communication 
```

Comment:
There's a typo here in "is to information regarding...."

Also, what about unwrapping everything so that it is easier to see what changes?